frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -15,35 +38,34 @@ export const getOCSRequestData = (
     },
     spec: {
       manageNodes: false,
-      storageDeviceSets: [
-        {
-          name: 'ocs-deviceset',
-          count: 1,
-          replica: 3,
-          resources: {},
-          placement: {},
-          portable: true,
-          dataPVCTemplate: {
-            spec: {
-              storageClassName: scName,
-              accessModes: ['ReadWriteOnce'],
-              volumeMode: 'Block',
-              resources: {
-                requests: {
-                  storage,
-                },
-              },
-            },
-          },
-        },
-      ],
+      storageDeviceSets: [createDeviceSet(scName, storage, true)],
```

Comment:
Its better  to pass the `provisioner` too as parameter in `createDeviceSet` for deciding the `portable` value instead of passing just `true`. This will help in case of multiple storage device sets creation.

Reply:
> Its better to pass the `provisioner` too as parameter in `createDeviceSet` for deciding the `portable` value instead of passing just `true`. 


Check again `createDeviceSet` is taking `portable` value after checking for provisioner.
The function takes in the value that re relevant for device set creation. 

> This will help in case of multiple storage device sets creation.

This is working